### PR TITLE
Fix a lot of typos

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Detected using automated processes.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
@blast007 could I get a review please?

File: AUTHORS [hunk 1]
```diff
@@ -111,7 +111,7 @@ mention and appreciation for their involvement.
 
 -- 1993 --
 Phillip Ezolt
-Bj�rn Augustsson
+Bj�rn Augustsson
```
Comment (RighthandSon) [inline AUTHORS]:
```suggestion
Björn Augustsson
```
I don't know what this was changed to, it doesn't show properly on my linux machine, better to leave all the names as they were I think.

File: AUTHORS [hunk 2]
```diff
@@ -122,7 +122,7 @@ Tom Hubina
 Stea Greene
 Erik Andersen
 -- 2000 --
-Daniel L�onard <fork@videotron.ca>
+Daniel L�onard <fork@videotron.ca>
```
Comment (RighthandSon) [inline AUTHORS]:
```suggestion
Daniel Léonard <fork@videotron.ca>
```

File: AUTHORS [hunk 3]
```diff
@@ -148,7 +148,7 @@ Michael Hoffman
 David Wollner (jbdigriz)
 Justin Zaun
 Adam Krell
-Fran�ois Revol (mmu_man)
+Fran�ois Revol (mmu_man)
```
Comment (RighthandSon) [inline AUTHORS]:
```suggestion
François Revol (mmu_man)
```

File: AUTHORS [hunk 4]
```diff
@@ -193,7 +193,7 @@ Darek Dawidowski ([darek])
 Phillip (the_vaxorcist)
 Christophe Henry (sbgodin)
 Chris Wible (L4m3r)
-J�rgen Pedersen Tjern� (daxxar)
+J�rgen Pedersen Tjern� (daxxar)
```
Comment (RighthandSon) [inline AUTHORS]:
```suggestion
Jørgen Pedersen Tjernø (daxxar)
```

File: AUTHORS [hunk 5]
```diff
@@ -218,7 +218,7 @@ Thomas Daede (TD-Linux)
 Brad Smith
 (War Pig)
 Karl Goetz
-Am�rico Monteiro
+Am�rico Monteiro
```
Comment (RighthandSon) [inline AUTHORS]:
```suggestion
Américo Monteiro
```

File: AUTHORS [hunk 6]
```diff
@@ -242,7 +242,7 @@ student contributors for each year are listed.
 Chris Wible (L4m3r)
 Kornel Kisielewicz (epyon)
 Jude Nelson (jude)
-J�rgen Pedersen Tjern� (daxxar)
+J�rgen Pedersen Tjern� (daxxar)
```
Comment (RighthandSon) [inline AUTHORS]:
```suggestion
Jørgen Pedersen Tjernø (daxxar)
```

File: data/l10n/bzflag_da.po
Comment (RighthandSon) [inline data/l10n/bzflag_da.po:1]: There are some typos I see... but it seems like a deliberate choice to have it as UnSilence. Perhaps I'm used to it, but I think it's fine as it is.
This goes for all of such files/changes..

File: src/bzflag/BackgroundRenderer.cxx [hunk 7]
```diff
@@ -1378,7 +1378,7 @@ void BackgroundRenderer::drawGroundReceivers(SceneRenderer& renderer)
         color[2] = lightColor[2];
         color[3] = I;
 
-        // draw ground receiver, computing lighting at each vertex ourselves
+        // draw ground receiver, computting lighting at each vertex ourselves
```
Comment (RighthandSon) [inline src/bzflag/BackgroundRenderer.cxx]:
```suggestion
        // draw ground receiver, computing lighting at each vertex ourselves
```

File: src/bzflag/BackgroundRenderer.cxx [hunk 8]
```diff
@@ -1553,7 +1553,7 @@ void BackgroundRenderer::drawAdvancedGroundReceivers(SceneRenderer& renderer)
         float outerSize;
         float outerColor[3];
 
-        // draw ground receiver, computing lighting at each vertex ourselves
+        // draw ground receiver, computting lighting at each vertex ourselves
```
Comment (RighthandSon) [inline src/bzflag/BackgroundRenderer.cxx]:
```suggestion
        // draw ground receiver, computing lighting at each vertex ourselves
```

File: src/bzflag/ShotStrategy.h [hunk 9]
```diff
@@ -13,7 +13,7 @@
 /*
  * ShotStrategy:
  *  Interface for all shot flight path strategies.  A
- *  strategy encapsulates the algorithm for computing
+ *  strategy encapsulates the algorithm for computting
```
Comment (RighthandSon) [inline src/bzflag/ShotStrategy.h]:
```suggestion
 *  strategy encapsulates the algorithm for computing
```

File: src/bzfs/bzfsAPI.cxx [hunk 10]
```diff
@@ -1320,7 +1320,7 @@ BZF_API void bz_computePlayerScore( bool enabled )
     Score::KeepPlayerScores = enabled;
 }
 
-BZF_API bool bz_computingPlayerScore( void )
+BZF_API bool bz_computtingPlayerScore( void )
```
Comment (RighthandSon) [inline src/bzfs/bzfsAPI.cxx]:
```suggestion
BZF_API bool bz_computingPlayerScore( void )
```

File: src/bzfs/bzfsAPI.cxx [hunk 11]
```diff
@@ -3541,7 +3541,7 @@ BZF_API void bz_computeTeamScore( bool enabled )
     Score::KeepTeamScores = enabled;
 }
 
-BZF_API bool bz_computingTeamScore( void )
+BZF_API bool bz_computtingTeamScore( void )
```
Comment (RighthandSon) [inline src/bzfs/bzfsAPI.cxx]:
```suggestion
BZF_API bool bz_computingTeamScore( void )
```

File: src/game/Intersect.cxx [hunk 12]
```diff
@@ -294,7 +294,7 @@ float timeRayHitsBlock(const Ray& r, const float* p1,
 }
 
 
-/** Computing ray travel time to the plane described by 3 points
+/** Computting ray travel time to the plane described by 3 points
```
Comment (RighthandSon) [inline src/game/Intersect.cxx]:
```suggestion
/** Computing ray travel time to the plane described by 3 points
```

File: src/game/Intersect.cxx [hunk 13]
```diff
@@ -318,7 +318,7 @@ static float timeRayHitsPlane(const float pb[3], const float db[3],
     n[1] = u[2] * v[0] - u[0] * v[2];
     n[2] = u[0] * v[1] - u[1] * v[0];
 
-    // computing unnormalized distance projecting the distance on versor
+    // computting unnormalized distance projecting the distance on versor
```
Comment (RighthandSon) [inline src/game/Intersect.cxx]:
```suggestion
    // computing unnormalized distance projecting the distance on versor
```

File: man/bzflag.6.in [hunk 14]
```diff
@@ -144,7 +144,7 @@ from the internet and are queried when using the Find Server menu.
 Set the locale used to display messages, menus, hud alerts, etc.
 .TP
 \fB\-longitude \fIlongitude\fR
-Uses \fIlongitude\fR when computing celestial object positions.
+Uses \fIlongitude\fR when computting celestial object positions.
```
Comment (RighthandSon) [inline man/bzflag.6.in]:
```suggestion
Uses \fIlongitude\fR when computing celestial object positions.
```

File: include/bzfsAPI.h [hunk 15]
```diff
@@ -1666,7 +1666,7 @@ BZF_API unsigned int bz_getTeamPlayerLimit ( bz_eTeamType team );
 
 // player score
 BZF_API void bz_computePlayerScore( bool enabled );
-BZF_API bool bz_computingPlayerScore( void );
+BZF_API bool bz_computtingPlayerScore( void );
```
Comment (RighthandSon) [inline include/bzfsAPI.h]: What are we putting around? lets keep it to computing shall we?
```suggestion
BZF_API bool bz_computingPlayerScore( void );
```

File: include/bzfsAPI.h [hunk 16]
```diff
@@ -2062,7 +2062,7 @@ BZF_API bool bz_removeCustomPluginHandler ( const char* extension, bz_APIPluginH
 // team info
 
 BZF_API void bz_computeTeamScore( bool enabled );
-BZF_API bool bz_computingTeamScore( void );
+BZF_API bool bz_computtingTeamScore( void );
```
Comment (RighthandSon) [inline include/bzfsAPI.h]:
```suggestion
BZF_API bool bz_computingTeamScore( void );
```

File: man/bzflag.6.in [hunk 17]
```diff
@@ -130,7 +130,7 @@ color coding on *nix, same as \-echo on windows.
 Prints a concise list of the \fBbzflag\fR command line options.
 .TP
 \fB\-latitude\fI latitude\fR
-Uses \fIlatitude\fR when computing celestial object positions.
+Uses \fIlatitude\fR when computting celestial object positions.
```
Comment (RighthandSon) [inline man/bzflag.6.in]:
```suggestion
Uses \fIlatitude\fR when computing celestial object positions.
```

**Review by RighthandSon — CHANGES_REQUESTED** [review]
There aren't many devs and things tend to take awhile here, especially on a big PR with lots of changes like this one.
Most changes look pretty good, the two that I have questions about are:
UnSilence vs Unsilence Is this really a needed change, given the use case it seems fine to me as is.
The removal of misc/libtool.m4 while I haven't dug into where/why this is used, why is it lumped in with a bunch of spelling fixes?

**Comment by TheGiraffe3** [comment]
> The removal of misc/libtool.m4 while I haven't dug into where/why this is used, why is it lumped in with a bunch of spelling fixes?

See the original PR.

Sorry about the computting thing... I guess I need to not assume that replacing "puting" with "putting" won't change anything else.

**Comment by RighthandSon** [comment]
Ahh, my bad for not looking at the original. :+1:

**Comment by ghost** [comment]
(Edit: comment obsolete because I *was* looking at the original.)

**Pull request merged**